man/systemd-analyze.xml

```diff
@@ -1109,6 +1109,21 @@ Service b@0.service not loaded, b.socket cannot be started.
       </varlistentry>
 
 
+      <varlistentry>
+        <term><option>--json=<replaceable>MODE</replaceable></option></term>
+
+        <listitem><para>With <command>security</command> command, generate a JSON formatted
+        output of the security analysis table. The format is a JSON array with objects
+        containing the following fields: set which indicates if the id has been set or not,
+        name which is what is used to refer to the id, json_field which is the JSON compatible
+        name of the id, description which is an outline of the id state, and exposure which
+        is an unsigned integer in the range 0.0..10.0, where a higher value corresponds to a
```

Comment:
you specify the range with one digit after the dot accuracy, but talk about integers. That's weird.